modules/core/nuxeo-core-convert/src/main/java/org/nuxeo/ecm/core/convert/extension/GlobalConfigDescriptor.java

```diff
@@ -21,38 +21,41 @@
 package org.nuxeo.ecm.core.convert.extension;
 
 import java.io.File;
-import java.io.IOException;
 
-import org.apache.commons.io.FileUtils;
 import org.nuxeo.common.Environment;
 import org.nuxeo.common.xmap.annotation.XNode;
 import org.nuxeo.common.xmap.annotation.XObject;
-import org.nuxeo.ecm.core.api.NuxeoException;
+import org.nuxeo.common.xmap.registry.XRegistry;
 
 /**
  * XMap Descriptor for the {@link org.nuxeo.ecm.core.convert.api.ConversionService} configuration.
  */
 @XObject("configuration")
+@XRegistry
 public class GlobalConfigDescriptor {
 
-    public static final boolean DEFAULT_CACHE_ENABLED = true;
-
     public static final long DEFAULT_GC_INTERVAL_IN_MIN = 10;
 
     public static final int DEFAULT_DISK_CACHE_IN_KB = 10 * 1024;
 
     public static final String DEFAULT_CACHING_DIRECTORY = "convertcache";
 
-    @XNode("enableCache")
-    protected Boolean enableCache;
-
-    public boolean isCacheEnabled() {
-        return enableCache == null ? DEFAULT_CACHE_ENABLED : enableCache.booleanValue();
-    }
+    @XNode(value = "enableCache")
+    protected boolean enableCache = true;
```

Comment:
Shouldn't this `true` be some kind of "defaultValue" (I don't remember the exact name)?

Reply:
defaultAssignment. I've kept this here as this descriptor is instantiated by the service when there is no configuration provided through XML.